src/change_stream.ts

```diff
@@ -609,6 +617,13 @@ export class ChangeStream<
    */
   static readonly RESUME_TOKEN_CHANGED = RESUME_TOKEN_CHANGED;
 
+  private timeoutContext?: TimeoutContext;
```

Comment:
Do we ever actually instantiate a timeout context?  I don't see it anywhere in the change stream.